FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/controller/ForwardAutonomousTest.java

```diff
@@ -36,6 +37,29 @@ public void testForwardMotion() {
 	}
 	
 	@Test
+	public void testGyroClockwiseForwardMotion() {
+		MockGyro gyro = new MockGyro();
+		gyro.gyroRate = 2;
+		AutoState auto = new ForwardAutonomous(1000, 1, gyro);
+		auto.init();
+		MotorOutput motorOutput = auto.driveTrainSpeed();
+		assertEquals(motorOutput.left, 1 , EPSILON);
+		assertEquals(motorOutput.right, 5.0 / 6.0, EPSILON);

```

Comment:
You're testing the wrong property, if it starts turning clockwise, it shouldn't get *more* clockwise.